src/System.Linq/src/System.Linq.csproj

```diff
@@ -23,6 +23,7 @@
       <Link>Common\System\NotImplemented.cs</Link>
     </Compile>
     <Compile Include="System\Linq\Enumerable.cs" />
+    <Compile Include="System\Linq\Enumerable.Sum.cs" />
```

Comment:
Yes, if we want to split the file, we should come up with a clear guidance/rule on how to do that and do that considering all operators, not just Sum.
It should be done as a separate change.